proposals/description/verify_rari_proposal.ts

```diff
@@ -0,0 +1,96 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const fip_x: TemplatedProposalDescription = {
+  title: 'Verify Rari deprecation proposal',
+  commands: [
+    {
+      target: 'rariUSDCPoolControllerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
+    {
+      target: 'rariDAIPoolControllerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
+    {
+      target: 'rariRSPTFundManagerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
+    {
+      target: 'rariDAIPoolManagerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
+    {
+      target: 'rariRSPTFundManagerProxy',
+      values: '0',
+      method: 'depositFees()',
+      arguments: (addresses) => [],
+      description: ''
+    },
+    {
+      target: 'rariDAIPoolManagerProxy',
+      values: '0',
+      method: 'depositFees()',
+      arguments: (addresses) => [],
+      description: ''
+    },
+    {
+      target: 'rariRSPTFundManagerProxy',
+      values: '0',
+      method: 'withdraw(string,uint256)',
+      arguments: (addresses) => ['USDC', '336235000000'],
+      description: ''
+    },
+    {
+      target: 'rariDAIPoolManagerProxy',
+      values: '0',
+      method: 'withdraw(string,uint256)',
+      arguments: (addresses) => ['DAI', '94218000000000000000000'],
+      description: ''
+    },
```

Comment:
are we sure these aren't higher because of the two fee collection actions above?

Reply:
they're correct, I already simulated the calculations